manim/cli/project/templates/Graph.mtp

```diff
@@ -0,0 +1,16 @@
+class GraphTemplate(GraphScene):
```

Comment:
We are deprecating GraphScene in #1161 soon and it would not be great to continue its usage in a template which should encourage best (new) practices. In which case, GraphTemplate should really be an AxisTemplate once #1161 is merged.

Reply:
Done